keyboards/flutterby/keymaps/default/keymap.c

```diff
@@ -0,0 +1,161 @@
+    },
+	[RAISE] = { /* Layer 1 */
+        {KC_EXLM,       KC_AT,   KC_HASH, KC_DLR,        KC_PERC,                   KC_EQL,        KC_7,          KC_8,    KC_9,    KC_0},
+	    {KC_CIRC,       KC_AMPR, KC_UNDS, KC_DQUO,       KC_QUOT,                   KC_SLSH,      KC_4,          KC_5,    KC_6,    KC_MINS},
+	    {KC_PIPE,       KC_BSLS, KC_TILD, KC_GRV,        KC_CALC,                   KC_ASTR,      KC_1,          KC_2,    KC_3,    KC_PLUS},
+	    {KC_TRNS,       KC_TRNS, KC_TRNS, LT(3,KC_TRNS), KC_TRNS,                   KC_TRNS,      KC_TRNS,       KC_0,    KC_DOT,  KC_TRNS}
+    },
+	[LOWER] = { /* Layer 2 */
+        {KC_VOLU,       KC_HOME, KC_UP,   KC_END,        KC_PGUP,                   KC_WBAK,      KC_WFWD,       KC_LPRN,   KC_RPRN,   KC_WH_U},
+	    {KC_VOLD,       KC_LEFT, KC_DOWN, KC_RGHT,       KC_PGDN,                   KC_NO,        KC_NO,         KC_LBRC,   KC_RBRC,   KC_WH_D},
+	    {KC_MUTE,       KC_CAPS, KC_SLCK, KC_INS,        KC_PSCR,                   KC_NO,        KC_NO,         KC_LCBR,   KC_RCBR,   KC_MYCM},
+	    {KC_TRNS,       KC_TRNS, KC_TRNS, KC_TRNS,       KC_TRNS,                   KC_TRNS,      LT(3,KC_TRNS), KC_TRNS, KC_TRNS, KC_TRNS}
+    },
+	[3] = { /* Function Layer */
+        {RGB_HUI,       RGB_SAI, RGB_SPI, RGB_VAI,       RGB_MOD,                   RESET,        KC_F7,         KC_F8,   KC_F9,   KC_F10},
+	    {RGB_HUD,       RGB_SAD, RGB_SPD, RGB_VAD,       RGB_RMOD,                  KC_SLEP,      KC_F4,         KC_F5,   KC_F6,   KC_F11},
+	    {RGB_M_P,       RGB_M_R, RGB_M_K, RGB_M_B,       RGB_TOG,                   LCA(KC_DEL),  KC_F1,         KC_F2,   KC_F3,   KC_F12},
+	    {KC_TRNS,       KC_TRNS, KC_TRNS, KC_TRNS,       KC_TRNS,                   KC_TRNS,      KC_TRNS,       KC_TRNS, KC_TRNS, KC_TRNS}
+    },
+	[4] = { /* Mouse Layer */
+		{KC_MUTE,       KC_WH_L,  KC_MS_U, KC_WH_R,       KC_WH_U,                   KC_BTN3,      KC_WH_U,       KC_VOLD, KC_VOLU, KC_MUTE},
+	 	{KC_VOLU,       KC_MS_L,  KC_MS_D, KC_MS_R,       KC_WH_D,                   KC_MS_L,      KC_MS_D,       KC_MS_U, KC_MS_R, KC_BTN1},
+		{KC_VOLD,       KC_BTN1,  KC_BTN2, KC_BTN3,       KC_FIND,                   KC_WH_L,      KC_WH_D,       KC_WH_R, KC_F11, KC_BTN2},
+		{KC_TRNS,       KC_TRNS,  KC_TRNS, KC_TRNS,       KC_TRNS,                   KC_TRNS,      KC_TRNS,       KC_TRNS, KC_TRNS, KC_TRNS}
+    }
+};
+
+
+void matrix_init_user(void) {
+}
+
+static const macro_t mac_cut[] PROGMEM = { D(LGUI), T(X), U(LGUI), END };
+static const macro_t win_cut[] PROGMEM = { D(LSFT), T(DELT), U(LSFT), END };
+
+static const macro_t mac_copy[] PROGMEM = { D(LGUI), T(C), U(LGUI), END };
+static const macro_t win_copy[] PROGMEM = { D(LCTL), T(INS), U(LCTL), END };
+
+static const macro_t mac_paste[] PROGMEM = { D(LGUI), T(V), U(LGUI), END };
+static const macro_t win_paste[] PROGMEM = { D(LSFT), T(INS), U(LSFT), END };
+
+const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt) {
```

Comment:
Could you please remove both the `action_get_macros`  and `fn_actions`, and convert these to the new style of macros? 

Also.... a lot of these could be replaced with `LSFT(KC_DEL)`, `LGUI(KC_X)` etc, so you don't need a lot of these. 

https://docs.qmk.fm/#/feature_macros